src/settings/routes/ExternalDataSourcesSettingsRoute.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import { stripesConnect } from '@folio/stripes/core';
+
+import ExternalDataSourcesForm from '../components/ExternalDataSourcesConfig/ExternalDataSourcesForm';
+
+class ExternalDataSourcesSettingsRoute extends React.Component {
+  static propTypes = {
+    resources: PropTypes.shape({
+      externalKbs: PropTypes.object,
+    }),
+    mutator: PropTypes.shape({
+      externalKbs: PropTypes.shape({
+        DELETE: PropTypes.func.isRequired,
+        POST: PropTypes.func.isRequired,
+        PUT: PropTypes.func.isRequired,
+      }),
+    }),
+  };
+
+  static manifest = Object.freeze({
+    externalKbs: {
```

Comment:
Can you sort this by `id;desc` so that when you create a new data source, it'll stay at the top of the list and not jump down to the bottom?